### DISCUSSION TITLE
Recommended Normalization Layer for GAT

### DISCUSSION THREAD

**Discussion by mertkiray** [pyg-team/pytorch_geometric#2732] [Q&A]
Hello, I know this is a dumb question but
Which normalization layer should I use default for GATs what is the current "to go" normalization layer? Should I go with Instance Norm, Layer Norm or Graph Norm?

**Comment by rusty1s** [comment]
I don't think there is any consensus on which normalization to use in combination for which layer. I actually find `BatchNorm` to work well in most cases, and I would only play around with the other options in case `BatchNorm` fails :)
